types/cuid/cuid-tests.ts

```diff
@@ -1,4 +1,5 @@
 
-import cuid = require('cuid')
+import * as cuid from 'cuid';
```

Comment:
Please revert - this is not a thing that works in ES6; the CJS-style import should be preferred